config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  root 'main#index'
+
+  resources :parks
+
+
+#Allows me to link from /views/main/index to /views/parks/index
+  get "/rails-national-parks/app/controllers/parks_controller.rb", to: "parks#index", as: "index"
```

Comment:
Clicking on the parks list link from your landing page is loading the file parks_controller.rb. get should be the route for the address bar not the file location. the to: "parks#index" is finding the file in the parks controller. Change the get to this:

get "/parks", to: "parks#index", as: "index"